src/views/Course.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <v-row class="main">
+    <v-col>
+      <v-sheet :height="managerHeight">
+        <h1>
+          {{ $route.params.id }}
+          <v-btn icon>
+            <v-icon class="mr-1"> mdi-download </v-icon>
+          </v-btn>
+          <v-btn icon>
+            <v-icon class="mr-1"> mdi-square-edit-outline </v-icon>
+          </v-btn>
+          <v-btn icon>
+            <v-icon class="mr-1"> mdi-delete </v-icon>
+          </v-btn>
+        </h1>
+        <router-link class="redirect" to="/manager">
+          <v-btn elevation="2" style="margin: 24px 0">Return to Manager</v-btn>
+        </router-link>
+        <v-btn elevation="2" style="margin: 24px" @click="onPickFile"
+          >Import Syllabus</v-btn
+        >
+        <input
+          type="file"
+          style="display: none"
+          ref="fileInput"
+          accept="application/pdf"
+          @change="onFilePicked"
+        />
+        <div>
+          <v-expansion-panels focusable>
+            <v-expansion-panel v-for="(item, i) in assessments" :key="i">
+              <v-expansion-panel-header>
+                <template>
+                  <v-row no-gutters>
+                    <v-col cols="1" class="assessment-item">
+                      {{ item.grade ? item.grade : 'N/A' }}
+                    </v-col>
+                    <v-col cols="7" class="assessment-item">
+                      {{ item.type }}: {{ item.description }}
+                    </v-col>
+                    <v-col cols="1" class="text--secondary assessment-item">
+                      <v-fade-transition leave-absolute>
+                        <span>
+                          {{ item.weight }}
+                        </span>
+                      </v-fade-transition>
+                    </v-col>
+                    <v-col cols="2" class="text--secondary assessment-item">
+                      <v-fade-transition leave-absolute>
+                        <span>
+                          {{
+                            item.deadline
+                              ? item.deadline
+                              : item.on_going
+                              ? 'On-going'
```

Comment:
Thoughts on the Nullish Coalescing operator here?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator